### ISSUE TITLE
Train a Custom Object Detection Model Using TensorFlow Object Detection API | MakeOptim

### ISSUE THREAD

**Issue by CatchZeng** [CatchZeng/makeoptim-gitalk#97]
https://makeoptim.com/en/deep-learning/yiai-object-detection 

 This article will introduce the concept of object detection, and explain how to use TensorFlow Object Detection API to train a custom object detector through cases, including data set collection and processing, TensorFlow Object Detection API installation, and model training.

**Comment by Tejatiriveedhi** [comment]
I am having issue with running the makefile and its giving me error with cp and -d, is this tutorial meant to work in Linux?

**Comment by CatchZeng** [comment]
> @Tejatiriveedhi
> I am having issue with running the makefile and its giving me error with cp and -d, is this tutorial meant to work in Linux?

Yes, it only works on Linux or Mac now. Because the official tutorial of TensorFlow Object Detection API is the same, see https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2.md.

In addition, I personally suggest that in machine learning, using Linux will avoid many problems, and many libraries can only run under Linux, such as https://github.com/magenta/magenta.

**Comment by Tejatiriveedhi** [comment]
> @CatchZeng
> > @Tejatiriveedhi
> > I am having issue with running the makefile and its giving me error with cp and -d, is this tutorial meant to work in Linux?
> 
> Yes, it only works on Linux or Mac now. Because the official tutorial of TensorFlow Object Detection API is the same, see https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2.md.
> 
> In addition, I personally suggest that in machine learning, using Linux will avoid many problems, and many libraries can only run under Linux, such as https://github.com/magenta/magenta.


Do you have any idea of how to run image segmentation algorithm using windows?

**Comment by CatchZeng** [comment]
> @Tejatiriveedhi
> > @CatchZeng
> > > @Tejatiriveedhi
> > > I am having issue with running the makefile and its giving me error with cp and -d, is this tutorial meant to work in Linux?
> > 
> > Yes, it only works on Linux or Mac now. Because the official tutorial of TensorFlow Object Detection API is the same, see https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2.md.
> > 
> > In addition, I personally suggest that in machine learning, using Linux will avoid many problems, and many libraries can only run under Linux, such as https://github.com/magenta/magenta.
> 
> 
> Do you have any idea of how to run image segmentation algorithm using windows?

Sorry, I don't have a Windows device on hand right now.

If you want to run it on Windows, you can try it out step by step according to [the official documentation of TensorFlow Object Detection API](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/tf2.md).

Another option is to install an Ubuntu on your Windows device.
